### ISSUE TITLE
Failure cross-compiling on macOS platform with rack-plugin-toolchain

### ISSUE THREAD

**Issue by cschol** [Paul-Dempsey/pachde1#8]
```
src/Info/Info.cpp:142:17: error: use of undeclared identifier 'isnanf'; did you mean 'isnan'?
            if (isnanf(font_size)) {                                                                                                                       
                ^~~~~~                                            
                isnan                                      
/home/build/rack-plugin-toolchain/local/osxcross/bin/../SDK/MacOSX11.1.sdk/usr/include/c++/v1/math.h:535:1: note: 'isnan' declared here
isnan(float __lcpp_x) _NOEXCEPT { return __libcpp_isnan(__lcpp_x); }
^                                
```

**Comment by Paul-Dempsey** [comment]
Fixed?  Use std::isnan instead., but I don't have a great way to verify at the moment.

d1be04be2e7173c161631e06ec51f99a3a7867bf

**Comment by Paul-Dempsey** [comment]
Really fixed : Paul-Dempsey/pachde1@620e1eeb8923a4051e4b2b13cdf1117c90230a0a
